scipy/stats/_sensitivity_analysis.py

```diff
@@ -272,22 +280,21 @@ def sobol_indices(
           (number of output variables), and
         - ``n`` is the number of samples (see `n` below).
 
-        Function evaluations values must be finite.
+        Function evaluation values must be finite.
 
         If `func` is a dictionary, contains the function evaluations from three
         different arrays. Keys must be: ``f_A``, ``f_B`` and ``f_AB``.
         ``f_A`` and ``f_B`` should have a shape ``(s, n)`` and ``f_AB``
         should have a shape ``(d, s, n)``.
-        This is an advanced feature and misuse can lead to wrong analysis.
+        This is an advanced feature and misuse can lead to incorrect analysis.
     n : int
-        Number of samples use to generate the matrices ``A`` and ``B``.
+        Number of samples used to generate the matrices ``A`` and ``B``.
         Must be a power of 2. The total number of points at which `func` is
         evaluated will be ``n*(d+2)``.
     dists : list(distributions), optional
```

Comment:
I prefer the original version as more explicit. "Should" could be just replaced with minimal adjustments

Reply:
I thought I was merely implementing what we had already discussed [here](https://discuss.scientific-python.org/t/discussion-of-sobol-indices-new-in-scipy-dev/756/4?u=tadamcz), so I'm not entirely sure what change you're looking for here.

I don't find this more explicit, but assuming that's what you want I've added back in

- the word "values"
- the sentence "The distribution of parameters depends on the application and should be carefully chosen"

If you still don't like the new version, can you just substitute in exactly what you'd like it to say?